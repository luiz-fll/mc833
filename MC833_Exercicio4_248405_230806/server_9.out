remoto: 127.0.0.1:6009
[SERVIDOR] Escutando em 127.0.0.1:6009
127.0.0.1:48308
GET / HTTP/1.0
Host: teste

127.0.0.1:48324
GET / HTTP/1.0
Host: teste

127.0.0.1:48338
GET / HTTP/1.0
Host: teste

127.0.0.1:48342
GET / HTTP/1.0
Host: teste

127.0.0.1:48358
GET / HTTP/1.0
Host: teste

127.0.0.1:48366
GET / HTTP/1.0
Host: teste

127.0.0.1:48368
GET / HTTP/1.0
Host: teste

127.0.0.1:48374
GET / HTTP/1.0
Host: teste

