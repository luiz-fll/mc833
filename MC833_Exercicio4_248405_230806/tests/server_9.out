remoto: 127.0.0.1:60009
[SERVIDOR] Escutando em 127.0.0.1:60009
127.0.0.1:51374
GET / HTTP/1.0
Host: teste

127.0.0.1:51376
GET / HTTP/1.0
Host: teste

127.0.0.1:51384
GET / HTTP/1.0
Host: teste

127.0.0.1:51388
GET / HTTP/1.0
Host: teste

127.0.0.1:51400
GET / HTTP/1.0
Host: teste

127.0.0.1:51404
GET / HTTP/1.0
Host: teste

127.0.0.1:51420
GET / HTTP/1.0
Host: teste

127.0.0.1:51430
GET / HTTP/1.0
Host: teste

127.0.0.1:51444
GET / HTTP/1.0
Host: teste

